### PR TITLE
expose CUDNN_BN_MIN_EPSILON from cudnn.h

### PR DESCRIPTION
**Pull request by kmaehashi**
Fix for https://github.com/chainer/chainer/issues/1960

### CODE REVIEW

File: cupy/cuda/cudnn.pyx [hunk 1]
```diff
@@ -995,6 +995,8 @@ cpdef poolingBackward(
 # Batch Normalization
 ###############################################################################
 
+CUDNN_BN_MIN_EPSILON = 1e-5
```
Comment (okuta) [inline cupy/cuda/cudnn.pyx]: How about fixing `cudnn.pxd` file?

Comment (kmaehashi) [reply cupy/cuda/cudnn.pyx]: Initially I tried the following code in pxd, but it does not work.

```py
cpdef double CUDNN_BN_MIN_EPSILON = 1e-5
```
Also, it cannot be enum as it is not integer.

Comment (okuta) [reply cupy/cuda/cudnn.pyx]: OK.

**Comment by okuta** [comment]
LGTM!

**Pull request merged**